sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/TimeSeriesInsightsClientSample/ModelSettingsSamples.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Threading.Tasks;
+using static Azure.IoT.TimeSeriesInsights.Samples.SampleLogger;
+
+namespace Azure.IoT.TimeSeriesInsights.Samples
+{
+    internal class ModelSettingsSamples
+    {
+        /// <summary>
+        /// This sample demonstrates getting Time Series model settings, updating a model and changing the default type Id for a model.
+        /// </summary>
+        public async Task RunSamplesAsync(TimeSeriesInsightsClient client)
+        {
+            PrintHeader("TIME SERIES INSIGHTS MODEL SETTINGS SAMPLE");
+
+            #region Snippet:TimeSeriesInsightsSampleGetModelSettings
+            Response<TimeSeriesModelSettings> getModelSettingsResponse = await client.ModelSettings.GetAsync();
+            Console.WriteLine($"Retrieved Time Series Insights model settings:\n{JsonSerializer.Serialize(getModelSettingsResponse.Value)}");
+            #endregion
+
+            // Store the default type Id so it can be used during clean up
+            string defaultTypeId = getModelSettingsResponse.Value.DefaultTypeId;
+
+            #region Snippet:TimeSeriesInsightsSampleUpdateModelSettingsName
+            Response<TimeSeriesModelSettings> updateModelSettingsNameResponse = await client.ModelSettings.UpdateNameAsync("NewModelSettingsName");
+            Console.WriteLine($"Updated Time Series Insights model settings name:\n" +
+                $"{JsonSerializer.Serialize(updateModelSettingsNameResponse.Value)}");
+            #endregion
+
+            // Create a Time Series type
+            var aggregateVariable = new AggregateVariable(new TimeSeriesExpression("count()"));
+            var variables = new Dictionary<string, TimeSeriesVariable>
+            {
+                { "aggregateVariableName", aggregateVariable },
+            };
+            var type = new TimeSeriesType("tsiTypeName", variables);
+            var timeSeriesTypes = new List<TimeSeriesType> { type };
+
+            Response<TimeSeriesTypeOperationResult[]> createTsiTypeResponse = await client.Types.CreateOrReplaceAsync(timeSeriesTypes);
+
+            // Ensure no error was reported as part of the response
+            if (createTsiTypeResponse.Value[0].Error != null)
+            {
```

Comment:
If there is an error should we continue?

Reply:
That's a good question. With TSI it's tricky because response will always be 200. I wanted to make sure I don't do any 'throws' or something that might pollute the snippet region because it won't make sense to the user looking at the snippet on Microsoft docs. But for this instance specifically, it does not belong to any snippet region. For that reason, I will wrap the success case in an 'if' statement.